app/views/users/index.html.erb

```diff
@@ -7,3 +7,6 @@
     <p>There's no users, please</p>
   <% end %>
 </section>
+<section class="add-section flex-column">
+  <%= link_to "Add a new post", new_user_post_path(@users) %>

```

Comment:
- [ ] Hi @henask12, though you added forms for creating posts, the error below on the screen record prevents a user from creating a post. Hence a user can't equally create a comment. Let's once more kindly double check and fix :wrench: the issue.

[Screencast from 10-06-2023 09:42:07 PM.webm](https://github.com/henask12/Blogify/assets/63087491/4c94571c-23f1-4d51-b0fd-12d5fb71c163)